### PR TITLE
Update drg module version. 

### PR DESCRIPTION
**Pull request by robo-cap**
Add support for custom existing DRG compartment id.

### CODE REVIEW

File: module-network.tf [hunk 1]
```diff
@@ -78,13 +78,14 @@ module "vcn" {
...
 module "drg" {
-  count          = tobool(var.create_drg) || var.drg_id != null ? 1 : 0
-  source         = "oracle-terraform-modules/drg/oci"
-  version        = "1.0.5"
-  compartment_id = coalesce(var.network_compartment_id, local.compartment_id)
+  count              = tobool(var.create_drg) || var.drg_id != null ? 1 : 0
+  source             = "oracle-terraform-modules/drg/oci"
+  version            = "1.0.6"
+  compartment_id     = coalesce(var.network_compartment_id, local.compartment_id)
+  drg_compartment_id = var.drg_compartment_id
```
Comment (hyder) [inline module-network.tf:85]: Does the `compartment_id` argument not do that already?

**Review by hyder — COMMENTED** [review]
Please see comment

File: module-network.tf (hunk 1 above)
Comment (robo-cap) [reply module-network.tf:85]: `compartment_id` - The compartment id where resources will be created.
`network_compartment_id` - The compartment id where network resources will be created.


This PR makes it possible to reference an existing DRG that is not in `compartment_id` or `network_compartment_id`.

**Review by hyder — APPROVED** [review]
Approved

**Comment by hyder** [comment]
I'll cut a patch release for you.

**Pull request merged**